scripts/ubuntu/mongo-orchestration.sh

```diff
@@ -1,5 +1,7 @@
 # Enable MongoDB Enterprise repo
 apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+# 3.2 key
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927
```

Comment:
Thanks. This jives with what I had to do for Travis CI in https://github.com/mongodb/mongo-php-driver/commit/b64561bfb0f245bc41fffbc17f062c678f887429.